aspnetcore/security/authentication/2fa.md

```diff
@@ -14,14 +14,14 @@ uid: security/authentication/2fa
 ---
 # Two-factor authentication with SMS
 
->[!WARNING]
-> This page documents version 1.0.0-beta8 and has not yet been updated for version 1.0.0
+> [!NOTE]
```

Comment:
Remove the note. We only add warnings when content is out of date. The assumption is that the document works with the current version.